seed/challenges/08-coding-interview-questions-and-take-home-assignments/coding-interview-data-structure-questions.json

```diff
@@ -192,9 +192,9 @@
     },
     {
       "id": "587d8255367417b2b2512c75",
-      "title": "How to Create a Circular Queue",
+      "title": "Create a Circular Queue",
       "description": [
-        "In this challenge you will be creating a Circular Queue. A circular queue is basically a queue that writes to the end of a collection then begins over writing itself at the beginning of the collection. This is type of data structure has some useful applications in certain situations. For example, a circular buffer can be used in streaming media. Once the queue is full, new media data simply begins to overwrite old data.",
+        "In this challenge you will be creating a Circular Queue. A circular queue is basically a queue that writes to the end of a collection then begins over writing itself at the beginning of the collection. This is type of data structure has some useful applications in certain situations. For example, a circular buffer can be used to stream media. Once the queue is full, new media data simply begins to overwrite old data.",
```

Comment:
I'd say "a circular queue is commonly used for streaming media" might be a better wording. That way, we're clearer about the use case, and avoid introducing a related word (but not the precise one we refer to in this challenge). What do you think? :blush: